protocol/river-status-unstable-v1.xml

```diff
@@ -47,7 +47,7 @@
     </request>
   </interface>
 
-  <interface name="zriver_output_status_v1" version="2">
+  <interface name="zriver_output_status_v1" version="3">
```

Comment:
It's actually necessary to bump this and `zriver_status_manager_v1` to version 4 for backwards compatibility. I'll fix this minor issue myself though during the rebase and merge.